src/app/styles/login.ts

```diff
@@ -0,0 +1,97 @@
+import styled from 'styled-components/native';
+
+export const Container = styled.View`
+    background-color: ${(props) => props.theme.colors.main};
+    flex: 1;
+`;
+export const Title = styled.Text`
+    color: ${(props) => props.theme.colors.secondary};
+    font-size: 48px;
```

Comment:
All numbers should be in the Default Theme interface and then we can use it like here:

font-size: ${(props) => props.theme.text.header}px;